landing_page/static/landing_page/js/search.js

```diff
@@ -1,5 +1,4 @@
-function loadDetailsPage(movie) {
+function loadDetailsPage(movie_id) {
     // movie = JSON.parse(jsonMovie);
-    console.log(movie);
-    window.location.href = window.location.origin + "/movie?movie=" + encodeURIComponent(JSON.stringify(movie));
+    window.location.href = window.location.origin + `/movie/${movie_id}`;
```

Comment:
this also doesn't work :(
![image](https://github.com/jonathanhouge/ReeldIn/assets/111781413/ca345642-cb14-42ac-9f2f-dc1779400499)